lang/test/org/partiql/lang/eval/evaluatortestframework/EvaluatorErrorTestCase.kt

```diff
@@ -65,15 +76,42 @@ data class EvaluatorErrorTestCase(
      */
     override val compilerPipelineBuilderBlock: CompilerPipeline.Builder.() -> Unit = { },
 
-    /**
-     * This will be executed to perform additional exceptions on the resulting exception.
-     */
-    val additionalExceptionAssertBlock: (SqlException) -> Unit = { }
 ) : EvaluatorTestDefinition {
 
     /** This will show up in the IDE's test runner. */
     override fun toString(): String {
         val groupNameString = if (groupName == null) "" else "$groupName"
         return "$groupNameString $query : $expectedErrorCode : $expectedErrorContext"
     }
+
+    /** A generated and human-readable description of this test case for display in assertion failure messages. */
+    fun testDetails(
```

Comment:
Should also include `target` in `testDetails` string

Reply:
done.